### ISSUE TITLE
Openobserve does not keep login session

### ISSUE THREAD

**Issue by pitastrudl** [openobserve/openobserve#4506]
### Which OpenObserve functionalities are the source of the bug?

login

### Is this a regression?

Yes

### Description

When logging into Openobserve, it logs out after not using it for a while or opening browser the next day. We have set the ZO_COOKIE_MAX_AGE: '2592000'.
In the browser I can see the cookie and it's age set properly. 
![image](https://github.com/user-attachments/assets/729e8d3f-abbf-4944-8a45-417f191038b4)
I have a few ideas but not sure what it is: 
1. Our loadbalancer setting, but this happens also in another environment with a F5 loadbalancer and our env with nginx loadbalancer.
2. We are referse proxying openobserve, maybe there is a setting affecting it? 
3. We use ZO_BASE_URI: "/openobserve", could this affect it?

I didnt manage to catch anything substantive in the logs. We have observed this issue since many major versions ago.

### Please provide the version you discovered this bug in (check about page for version information)

```true
v0.11.0
7bbe192c015640ce643364b83b8ebb9c76a13331
2024-09-02T08:02:25Z
```

**Comment by pitastrudl** [comment]
Forgot to add, when logging back in and while I am at the login screen, I can see in the browser storage that the cookie is there.

**Comment by hengfeiyang** [comment]
Yep, here just using the cookie expired time, nothing special. please confirm that:
<img width="1850" alt="image" src="https://github.com/user-attachments/assets/64321e2f-9875-4430-91cc-f29689c0f43d">

**Comment by pitastrudl** [comment]
Hi, this is what i get when i try to log in, even though I logged in a few days ago. Is this what was to confirm?
![image](https://github.com/user-attachments/assets/7c1f7a75-302f-422d-bf19-a2df4587b707)

**Comment by hengfeiyang** [comment]
sorry for late, can you check it again?

**Comment by pitastrudl** [comment]
Hey, no problem, I'll check in the coming days. Which version should I try? Latest stable release seems to be v0.14.1.

**Comment by hengfeiyang** [comment]
This shouldn't be a problem now.
